src/test/java/com/notably/logic/parser/NotablyParserTest.java

```diff
@@ -94,13 +95,45 @@ public void parseCommand_openCommandInput_openCommand() throws Exception {
         assertEquals(AbsolutePath.fromString("/another/CS2103"), model.getCurrentlyOpenPath());
     }
 
+    @Test
+    public void parseCommand_openCommandInputNoPrefix_openCommand() throws Exception {
+        Command command = parser.parseCommand("open CS2104").get(0);
+
+        assertTrue(command instanceof OpenCommand);
+
+        command.execute(model);
+
+        assertEquals(AbsolutePath.fromString("/another/CS2103"), model.getCurrentlyOpenPath());
+    }
+
     @Test
     public void parseCommand_deleteCommandInput_deleteCommand() throws Exception {
         Command command = parser.parseCommand("delete -t ../CS2103").get(0);
 
         assertTrue(command instanceof DeleteCommand);
     }
 
+    @Test
+    public void parseCommand_deleteCommandInputNoPrefix_deleteCommand() throws Exception {
+        Command command = parser.parseCommand("delete ../CS2103").get(0);
+
+        assertTrue(command instanceof DeleteCommand);
+    }
+
+    @Test
+    public void parseCommand_editCommandInput_editCommand() throws Exception {
+        Command command = parser.parseCommand("edit -b ../CS2103").get(0);
+
+        assertTrue(command instanceof EditCommand);
+    }
+
+    @Test
+    public void parseCommand_editCommandInputNoPrefix_editCommand() throws Exception {
+        Command command = parser.parseCommand("edit ../CS2103").get(0);
+
+        assertTrue(command instanceof EditCommand);
+    }
```

Comment:
Edit's `-b` flag should indicate body right? Could this be a typo?

Reply:
Oh yes, i've made the necessary changes!